cabal-install/src/Distribution/Client/Errors.hs

```diff
@@ -0,0 +1,227 @@
+{-# LANGUAGE ConstraintKinds #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE InstanceSigs #-}
+
+-----------------------------------------------------------------------------
+
+-- Module      :  Distribution.Client.Errors
+-- Copyright   :
```

Comment:
These should probably be filled in somehow.

Reply:
I think this can only be filed with @SuganyaAK's name, because we do not do the paperwork of copyright transfer. However, another option is to just delete the field. Cabal is distributed under BSD3 and for that license, the practice of putting copyright in each file is not common (as opposed to GNU GPL, which do recommend it in their very text).

Reply:
I have filled in with my name and matched the rest with other similar files.
@Mikolaj Thanks for the comment.